src/platform/nxp/common/OTAImageProcessorImpl.h

```diff
@@ -77,6 +78,8 @@ class OTAImageProcessorImpl : public OTAImageProcessorInterface
 
     /* Buffer used for transaction storage */
     uint8_t mPostedOperationsStorage[NB_PENDING_TRANSACTIONS * TRANSACTION_SZ];
+
+    uint16_t delayBeforeReboot = 0;
```

Comment:
add units to variables. Also members use `m` in CHIP as a prefix. This should probably be `mDelayBeforeRebootMs` or something like that.